server/services/group.js

```diff
@@ -31,4 +31,33 @@ function updateGroup(groupName, groupObject) {
       Group.findOneAndUpdate({ name: groupName }, validatedGroupObject, { new: true }));
 }
 
-export default { getGroup, deleteGroup, createGroup, updateGroup };
+function addUser(group, userId) {
+  const userIdSchema = Joi.string().hex().length(24);
+
+  return JoiHelper.validate(userId, userIdSchema)
+    .then((validatedUserObject) => {
+      group.addUser(validatedUserObject);
+      return group;
```

Comment:
Because `group.addUser(validatedUserObject);` returns the group object you can just do this 
`.then((validatedUserObject) => group.addUser(validatedUserObject));`

Same goes for the removeUser() {}